web/wp-content/themes/dctx/analyticstracking.js

```diff
@@ -0,0 +1,7 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
```

Comment:
:warning: **Warning**: Found precision alignment of 2 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).